README.md

```diff
@@ -2,84 +2,80 @@
 
 <!--- insert badge here --->
 
-[CU Get Reg](https://cugetreg.com) เว็บไซต์ที่จะช่วยอำนวยความสะดวกให้กับนิสิตจุฬาลงกรณ์มหาวิทยาลัยทุกคนในการค้นหาวิชาเรียนและวางแผนจัดตารางเรียนล่วงหน้าก่อนการลงทะเบียนจริง พัฒนาโดยชมรม [Thinc.](https://thinc.in.th)
+Have no idea what courses to take? Don't know what this course is about? We got it covered. With [CU Get Reg](https://cugetreg.com), we put all kinds of course data together so that you don't have to.
 
-## สารบัญ
+## Contents
 
-- [จุดเริ่มต้นของ CU Get Reg](#beginning-of-cu-get-reg)
-- [โครงการที่เกี่ยวข้อง](#related-repositories)
-- [มาเป็นส่วนหนึ่งกับเรา](#contribution)
+- [The beginning of CU Get Reg](#beginning-of-cu-get-reg)
+- [Related repositories](#related-repositories)
+- [Contribution](#contribution)
 - [Roadmap](#roadmap)
-- [ทีมผู้พัฒนา](#developers)
-- [ติดต่อเรา](#contacts)
-- [ผู้สนับสนุน](#sponsors)
+- [Developers](#developers)
+- [Contacts](#contacts)
+- [Sponsors](#sponsors)
 
 <div id="beginning-of-cu-get-reg"></div>
 
-## จุดเริ่มต้นของ CU Get Reg
+## The beginning of CU Get Reg
 
-    "GenEd เทอมนี้เราจะลงวิชาอะไรดีนะ?"
+    "Which GenEd course do I enroll this semester?"
 
-    "GenEd ที่เราเลือกลงไปเวลาจะไปทับกับวิชาบังคับหรือเปล่านะ?"
+    "Will this GenEd course conflict with my compulsary courses?"
```

Comment:
```suggestion
    "Will this course conflict with my other courses?"
```
I don't think this applies to just GenEds, and I want people to be aware that CUgetreg focuses on more than just GenEds